test/Microsoft.AspNet.Server.KestrelTests/EngineTests.cs

```diff
@@ -116,6 +116,22 @@ public void ConnectionCanReadAndWrite(TestServiceContext testContext)
 
             Console.WriteLine("Started");
             var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+            if (PlatformApis.IsWindows)
+            {
+                const int SIO_LOOPBACK_FAST_PATH = (-1744830448);
+                byte[] optionInValue = BitConverter.GetBytes(1);
+                try
+                {
+                    socket.IOControl(SIO_LOOPBACK_FAST_PATH, optionInValue, null);
+                }
+                catch (Exception e)
```

Comment:
In the code provided on TechNet in the Server & Tools blog which seems to be the main detail around the announcement of this feature they are catching `SystemException`; however this exception type does seem to exist in coreclr. Hazarding a guess it would be a `SocketException` with `ErrorCode` 10045